dressup/src/pages/ProductsPage/ProdListingPage/reducerFn.js

```diff
@@ -0,0 +1,32 @@
+export const reducerFn = (state, action) => {
+  switch (action.type) {
+    case "highToLow":
+      return { ...state, sortOrder: action.value };
+    case "lowToHigh":
+      return { ...state, sortOrder: action.value };
+    case "FiveStars":
+      return { ...state, rating: action.value };
+    case "FourStars":
+      return { ...state, rating: action.value };
+    case "ThreeStars":
+      return { ...state, rating: action.value };
+    case "TwoStars":
+      return { ...state, rating: action.value };
+    case "OneStar":
+      return { ...state, rating: action.value };
+    case "All":
```

Comment:
Suggestion: you can here give one type for rating like "FILTER_BY_RATINGS` and can sort by `action.value`